### PR TITLE
Use `following` and `followers` scopes in CLI

### PR DESCRIPTION
**Pull request by mjankowski**

### CODE REVIEW

File: lib/mastodon/cli/accounts.rb [hunk 1]
```diff
@@ -472,15 +472,13 @@ def reset_relationships(username)
       end
 
       total     = 0
-      total    += Account.where(id: ::Follow.where(account: account).select(:target_account_id)).count if options[:follows]
-      total    += Account.where(id: ::Follow.where(target_account: account).select(:account_id)).count if options[:followers]
+      total    += follows_for(account).count if options[:follows]
+      total    += followers_for(account).count if options[:followers]
```
Comment (ClearlyClaire) [inline lib/mastodon/cli/accounts.rb:476]: I'm wondering why this has been written this way in the first place, and shouldn't this just be `account.following` and `account.followers`?
Having a look, the SQL queries are slightly different, but the query plans are exactly the same.

File: lib/mastodon/cli/accounts.rb [hunk 2]
```diff
@@ -493,9 +491,7 @@ def reset_relationships(username)
       end
 
       if options[:followers]
-        scope = Account.where(id: ::Follow.where(target_account: account).select(:account_id))
-
-        scope.find_each do |target_account|
+        followers_for(account).find_each do |target_account|
```
Comment (ClearlyClaire) [inline lib/mastodon/cli/accounts.rb:494]: Same, `account.following.find_each` and `account.followers.find_each` result in similar but different SQL queries with identical query plans.

**Review by ClearlyClaire — CHANGES_REQUESTED** [review]
As explained in inline commands, I think this should just use `#following` and `#followers` instead of redefining them.

File: lib/mastodon/cli/accounts.rb [hunk 3]
```diff
@@ -651,6 +647,14 @@ def migrate(username)
 
     private
 
+    def unordered_follows_for(account)
+      account.following.reorder(nil)
+    end
+
+    def unordered_followers_for(account)
+      account.followers.reorder(nil)
+    end
```
Comment (ClearlyClaire) [inline lib/mastodon/cli/accounts.rb:656]: I honestly don't think adding one level of indirection with dedicated methods for this is worth it, `account.following.reorder(nil)` is already descriptive enough imho.

Comment (mjankowski) [reply lib/mastodon/cli/accounts.rb]: Fair enough, pushed again to keep a little repetition, but use the associations directly.

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**